src/calculateRentalCost.js

```diff
@@ -3,8 +3,21 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
   // write code here
+  const dayCost = 40;
+  const discount3 = 20;
+  const discount7 = 50;
+  const totalAmount = days * dayCost;
+
+  if (days < 3) {
```

Comment:
Consider renaming 'days' to 'numberOfDays' for increased clarity in expressing the purpose of the variable.